lib/image-stream/handlers.js

```diff
@@ -58,7 +58,7 @@ module.exports = {
           value: options.size
         }
       },
-      transform: Bluebird.resolve(unbzip2Stream())
+      transform: unbzip2Stream()
```

Comment:
Oh, for some reason I had the impression that we needed to resolve a value here.